src/api/profile.ts

```diff
@@ -137,3 +137,27 @@ export const updatePersonalizedSchedule = async ({
     });
   });
 };
+
+export interface SavePosterToProfileProps {
+  venueId: string;
+  userId: string;
+  removeMode?: boolean;
+}
+
+export function savePosterToProfile({
+  venueId,
+  userId,
+  removeMode = false,
```

Comment:
I've refactored as requested here. Thanks!

Reply:
@margulies thank you for your update.
But next time, could you attach the commit URL where the suggestions from the comment were fixed? It will make the reviewer's life a bit easier during the next round of the review.

refactored in https://github.com/sparkletown/sparkle/pull/1375/commits/2ff03724dd3b34e1b988a449566fed0dfc929c05